c10/core/DispatchKey.cpp

```diff
@@ -149,8 +152,6 @@ const char* toString(DispatchKey t) {
     // https://github.com/zou3519/functorch
     // We plan on eventually upstreaming the prototype into core, at which
     // point it will have a different design that should use fewer keys.
-    case DispatchKey::FuncTorchPython:
```

Comment:
The comment above should go?

Reply:
yup

Reply:
actually not sure if the comment above is still valid for the keys below. cc. @zou3519

Reply:
Please keep the comment